packages/host/app/components/operator-mode/profile-info-popover.gts

```diff
@@ -22,68 +28,202 @@ interface ProfileInfoSignature {
+        padding-top: var(--boxel-sp-lg);
+        border-top: 1px solid var(--boxel-dark);
+      }
+      .info-group {
+        display: flex;
+        flex-direction: column;
+        gap: var(--boxel-sp-xxs);
+      }
+      .label {
+        color: var(--boxel-dark);
+        font: var(--boxel-font-xs);
+      }
+      .out-of-credit {
+        font: var(--boxel-font-sm);
+      }
+      .info-group .value {
         color: var(--boxel-dark);
-        text-transform: uppercase;
+        font: 700 var(--boxel-font-sm);
+        font-style: capitalize;
       }
     </style>
 
     <div class='profile-popover' data-test-profile-popover ...attributes>
       <header class='header'>
-        <div class='label'>
-          Signed in as
-        </div>
-
         <BoxelButton
           @kind='secondary-light'
-          @size='extra-small'
+          @size='small'
           {{on 'click' @toggleProfileSettings}}
           data-test-settings-button
         >
           Settings
         </BoxelButton>
+
+        <BoxelButton
+          @kind='primary-dark'
+          @size='small'
+          {{on 'click' this.logout}}
+          data-test-signout-button
+        >
+          Sign Out
+        </BoxelButton>
       </header>
 
       <ProfileInfo />
-
-      <BoxelButton
-        {{on 'click' this.logout}}
-        @kind='primary-dark'
-        data-test-signout-button
-      >
-        Sign out
-      </BoxelButton>
+      {{! TODO: Remove config.APP.creditInfoEnabled once the API integration for credit info is completed. }}
+      {{#if config.APP.creditInfoEnabled}}
+        <div class='credit-info' data-test-credit-info>
+          <div class='info-group'>
+            <span class='label'>Membership Tier</span>
+            <span
+              class='value'
+              data-test-membership-tier
+            >{{this.plan.name}}</span>
+          </div>
+          {{#if this.isChangePlanDisplayed}}
+            <BoxelButton
+              @kind='secondary-light'
+              @size='small'
+              data-test-change-plan-button
+              {{on 'click' @toggleProfileSettings}}
+            >Change plan</BoxelButton>
+          {{/if}}
+          <div class='info-group'>
+            <span class='label'>Monthly Credit</span>
+            <span
+              class='value'
+              style={{if this.isOutOfPlanCreditAllowance 'color:#ff3838'}}
+              data-test-monthly-credit
+            >{{this.monthlyCreditText}}</span>
+          </div>
+          {{#if this.isOutOfCredit}}
+            <span class='out-of-credit' data-test-out-of-credit>You have used
+              all your credits from your monthly plan. Please upgrade your plan
+              or buy additional credit to continue.</span>
+          {{/if}}
+          <div class='info-group'>
+            <span class='label'>Additional Credit</span>
+            <span class='value' data-test-additional-balance>{{if
+                this.isFreePlan
```

Comment:
If a user is downgraded to a free plan, and has extra credits, how can the user see how many extra credits are available?

Reply:
Could we just add it to the monthly credits? If a user downgrades from the Creator plan to the Free plan with 1,000 credits remaining, the total monthly credits would be 2,000 (1,000 from the Free plan and 1,000 from extra credits). This way, we can maintain the design. @lukemelia, we need your thoughts on this.

Reply:
Note: We are still waiting for the final design.